domdiv/main.py

```diff
@@ -971,17 +972,36 @@ def filter_sort_cards(cards, options):
     if options.expansion_dividers:
 
         cardnamesByExpansion = {}
+        randomizersByExpansion = {}
         for c in cards:
             if cardSorter.isBaseExpansionCard(c):
                 continue
-            cardnamesByExpansion.setdefault(c.cardset, []).append(c.name.strip().replace(' ', '&nbsp;'))
+            if c.randomizer:
+                randomizersByExpansion[c.cardset] = randomizersByExpansion.setdefault(c.cardset, 0) + 1
+            else:
+                randomizersByExpansion[c.cardset] = randomizersByExpansion.setdefault(c.cardset, 0)
+
+            if c.cardset not in cardnamesByExpansion:
+                cardnamesByExpansion[c.cardset] = {}
+            if c.card_tag not in cardnamesByExpansion[c.cardset]:
+                # Save off information about the card to be used on the expansion divider
+                order = 0
+                if c.card_tag in cardSorter.baseOrder:
+                    # Use the base card ordering
+                    order = 100 + cardSorter.baseOrder.index(c.card_tag)
+                cardnamesByExpansion[c.cardset][c.card_tag] = {'name': c.name.strip().replace(' ', '&nbsp;'),
+                                                               'randomizer': c.randomizer,
+                                                               'count': 1,
+                                                               'sort': "%03d%s" % (order, c.name.strip(),)}
+            else:
+                cardnamesByExpansion[c.cardset][c.card_tag]['count'] += 1
```

Comment:
Maybe I don't totally understand the intent here - how is this count ever > 1? Don't we always have a single instance of each card name per expansion? At least when I run this this is the case - or is it in combination with some other option that I'm not thinking about?

Reply:
Ahhh, well I cheated.  It can be greater than one if you break up Curses or add starting decks.